### ISSUE TITLE
Remove build results /dist and template.html

### ISSUE THREAD

**Issue by psolbach** [liveblog/liveblog-default-theme#5]

**Comment by mscam** [comment]
Due to backward incompatibility with current backend server, we cannot remove dist directory and template.html. Theme build process is currently not performed by backend server.

**Comment by psolbach** [comment]
Exactly! We should put the "pre-built" theme somewhere else. We will need the bundled assets on a CDN anyway for people to extend from.
